use-cases/NIC-passthrough-and-Kata.md

```diff
@@ -0,0 +1,292 @@
+# How To Pass a Physical NIC Into a Kata Container
+![](https://github.com/Yuval-Ai/documentation/blob/doc-create-NIC-passthrough-and-Kata/use-cases/images/NIC%20passthrough%20Diagram.png)
+
+
+In this guide we walk through the process of passing a physical NIC into a KC (Kata Container). The "usual" way that a KC is wired for communication is through a series of bridges and virtual NICs as can be seen in the Image above (marked as default). 
+
+For some use cases, the container demands a direct link to the physical port of the host device, for example in a situation were the container is required to route high BW traffic without having support for acceleration such as SR-IOV.
+
+#### In this guide:
+
+* [Before you start (Restrictions, Requirements and Assumptions)](#before-you-start-restrictions-requirements-and-assumptions)
```

Comment:
I think we normally just place this at the top of the document as a TOC in Kata docs. Have a look at the [kata documentation guide](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md) if you have not already - there are probably some other conventions we use that will apply to the rest of the document.

Maybe also see something like the existing [GPU guide](https://github.com/kata-containers/documentation/blob/master/use-cases/Intel-GPU-passthrough-and-Kata.md) for an example of style and flow.